README.md

```diff
@@ -215,6 +219,32 @@ script:
       bash <(curl -s https://codecov.io/bash) -f lcov.info;
 ```
 
+### grcov with GitLab CI and codecov.io
+
+```yaml
+codecov:
+  stage:                           coverage
+  variables:
+    CARGO_INCREMENTAL:             0
+    RUSTFLAGS:                     "-Zprofile -Zmir-opt-level=0 -Ccodegen-units=1
+                                      -Copt-level=0 -Clink-dead-code -Coverflow-checks=off"
+    # We removed the `-Cinline-threshold=0` flag from the above `RUSTFLAGS` since it was bugged
```

Comment:
This is no longer in the README, since 2d37c3aa6d9ddca95fd95e695fc925bfb428745d

Reply:
removed